lib/cluster.js

```diff
@@ -34,8 +34,7 @@ function Worker(options) {
   if (options.process) {
     this.process = options.process;
     this.process.on('error', (code, signal) =>
-      this.emit('error', code, signal)
-    );
+      process.nextTick(() => this.emit('error', code, signal)));
```

Comment:
IMHO it's better to leave the `);` on the next line to make it easier to see where it ends.